CHANGES/1639.misc

```diff
@@ -0,0 +1 @@
+Add exporter command for project wisdom
```

Comment:
since this PR is linked to issue AAH-1624, this filename will need to be renamed to `1624.misc` to pass ci

Reply:
sorry, i read `Issue: AAH-1624` in the PR description and i thought that was the issue key, but i see from the commit message `Issue: AAH-2343` that is the correct issue key. so you'll want 1 file in `CHANGES` called `2343.misc`